seed/challenges/basejumps.json

```diff
@@ -102,8 +102,93 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Prepárate para los Basejumps",
+      "descriptionEs": [
+        [
+          "http://i.imgur.com/4IZjWZ3.gif",
+          "Una imagen gif que te muestra cómo crear una cuenta en c9.io.",
+          "Te recomendamos resolver nuestros desafíos de full stack en c9.io, un poderoso ambiente de desarrollo basado en tu navegador. Esto te ahorrará muchas horas que utilizarías configurando tu computadora para correr Node.js y MongoDB - tiempo que podrías utilizar escribiendo código. <br>Crea una cuenta en c9.io pulsando el símbolo de GitHub en la esquina superior derecha de la página de c9.io. Pulsa el botón con el símbolo de suma para crear una área de trabajo nueva. Introduce tu dirección de correo electrónico cuando se te solicite.",
+          "http://c9.io"
+        ],
+        [
+          "http://i.imgur.com/F7i5Hhi.gif",
+          "Una imagen gif que te muestra cómo llenar el formulario para crear un área de trabajo nueva",
+          "En vez de iniciar desde cero, recomendamos utilizar Clementine.js, un modelo (\"boilerplate\") de JavaScript full stack que viene con código básico ya escrito para ti. Clementine.js tiene un tutorial detallado que puedes seguir para construirlo por ti mismo, pero por ahora simplemente vamos a clonarlo. En c9.io, dale un nombre a tu área de trabajo, luego deja \"Plantilla\" (\"Template\") como personalizado y crea tu espacio de trabajo usando el siguiente url de Github: <code>https://github.com/johnstonbl01/clementinejs-fcc.git</code>",
```

Comment:
Bien por traducción entre paréntesis a continuación de término en español usado.  Me parece que sobran las comillas (ya hay paréntesis), pero me parece que podemos unificar en estilo que pongamos esos términos en itálicas, intenta con &lt;em>bolierplate&lt;/em>